Website/DotNetNuke.Website.csproj

```diff
@@ -3288,6 +3288,7 @@
     <Content Include="Providers\DataProviders\SqlDataProvider\09.02.00.SqlDataProvider" />
     <Content Include="Providers\DataProviders\SqlDataProvider\09.02.01.SqlDataProvider" />
     <Content Include="Providers\DataProviders\SqlDataProvider\09.02.02.SqlDataProvider" />
+    <Content Include="Providers\DataProviders\SqlDataProvider\09.02.02.01.SqlDataProvider" />
```

Comment:
This should be 09.03.00.01

Reply:
I can rename that. Just need to figure out how my local build-to-test process works after that.

Reply:
IMO we don't care whether it is named 09.03.00.sqldataprovider or 09.03.00.01.sqldataprovider, though I prefer am the second option, reserving the first of upgrade of version numbers.
Once the SQL files are restructured, we may omit the subversion files.